Contrib/README.MSXPi

```diff
@@ -0,0 +1,155 @@
+Architecture of the MSXPi extension for openMSX: 
+┌────────────┐         ┌────────────────────────┐         ┌────────────────────┐
+│    MSX     │  <--->  │     MSXPi Interface    │  <--->  │   msxpi-server     │
+│ (Z80 I/O)  │  0x56   │ MSXPiDevice + socket)  │  Socket │(Python socket API) │
+└────────────┘  0x5A   └────────────────────────┘         └────────────────────┘
+
+Now that you know everything you need (not really!) to start with MSXPi, prepare
+the environment.
+
+To use this extension, you will need Python3 installed in your computer.
+IDLE will work with MSXPi in Windows - it can be downloaded and installed
+from Microsoft App store. MSXPi will also work in other platforms - it
+works better on Linux, since it is the native environment it was designed
+for initially.
+At this time, some commands will work on Windows, many won't - but they
+are being ported and pushed to MSXPi repository, so I recommend to keep
+refreshing your local copy frequently.
+
+There are a lot of Python modules that are not standard in Python, and will
+require you to install before running MSXPi. When you run msxpi-server, it
+will complain about missing modules - use that information to install them.
+For example, to install module "fs":
+
+Go to Windows command line and type:
+
+   python -m pip install fs
+
+Repeat for all modules that are not already installed.
+
+Some commands (such as ploadr) will try to use 7-Zip, LHA or UNAR to 
+decompress files.. You will need to install those tools also or the
+command will fail.
+
+You will also have to download several files needed by MSXPi - follow the 
+instructions below to get started.
+
+Steps to Setup and Try MSXPi in openMSX
+=======================================
+
+1. Install an openMSX version that has support for MSXPi extension. You can download
+   binaries for all platforms (nightly builds from openMSX) in the releases section:
+   https://github.com/costarc/openMSX/releases
+   
+   Always use the latest release.
+   
+2. Download msxpi-server.py (the server component that runs in Python) from:
+   https://github.com/costarc/MSXPi/tree/master/software/Server/Python/src
+   
+   Run from IDLE console, or from command line:
+   
+   python msxpi-server.py
+   
+3. Download msxpibios.rom from:
+   https://github.com/costarc/MSXPi/tree/master/software/openMSX/share/systemroms/extensions
+   
+   Copy to you openMSX/share/systemroms/extensions/ folder.
+   
+4. Download the matching MSXPi.xml from:
+   https://github.com/costarc/MSXPi/tree/master/software/openMSX/share/extensions
+   
+   copy to you openMSX/share/extensions/ folder.
+
+5. Use a DIR as DISK with openMSX to facilitate the tests.
+   Create a directory named FloppyA
+   Copy MSXDOS.SYS and COMMAND.COM to FloppyA
+   Make sure openMSX can boot into MSX-DOS using this folder as a drive:
+   
+   openmsx -machine Panasonic_FS-A1WSX -diska FloppyA
+   
+6. Dowload the MSXPi commands you want to test from:
```

Comment:
Typo: Dowload -> download

I noticed this by accident, I did not do a full spelling check ;-)
But maybe it's a good idea to do that.